flake.nix

```diff
@@ -0,0 +1,379 @@
+              doCheck = false;
+            };
+
+            staticfiles = pkgs.buildGoModule {
+              pname = "staticfiles";
+              version = "0.0.1"; # no official version
+              src = pkgs.fetchFromGitHub {
+                owner = "isubasinghe";
+                repo = "staticfiles";
+                rev = "3d5ddde4d52ddef391b5f4f37e06c80980a5c0c2";
+                sha256 = "sha256-fyamqYhKXnY0fzNhS3SL15yHDA/pIuoQ+NsehfA7BCE=";
+              };
+              vendorHash = null;
+            };
+            default = config.packages.${package.name};
+          };
+
+          devShells = {
+            ${package.name} = pkgs.mkShell {
+              inherit (package) name;
+              shellHook = ''
+                unset GOPATH;
+              '';
+              inputsFrom = [ config.packages.${package.name} ];
+              packages = with pkgs; [
+                config.packages.mockery
+                config.packages.protoc-gen-gogo-all
+                config.packages.grpc-ecosystem
+                config.packages.go-swagger
+                config.packages.controller-tools
+                config.packages.k8sio-tools
+                config.packages.goreman
+                config.packages.stern
+                config.packages.staticfiles
+                config.packages.${package.name}
+                nodePackages.shell.nodeDependencies
+                gopls
+                go
+                jq
+                nodejs
+                pythonEnv
+                clang-tools
+                protobuf
+                myyarn
+              ];
+            };
+
+            devEnv = inputs.devenv.lib.mkShell {
+              inherit inputs pkgs;
+              modules = [
+                ({ pkgs, ... }: {
+                  env = argoConfig.env;
+                  # This is your devenv configuration
+                  packages = with pkgs; [
+                    config.packages.mockery
+                    config.packages.protoc-gen-gogo-all
+                    config.packages.grpc-ecosystem
+                    config.packages.go-swagger
+                    config.packages.controller-tools
+                    config.packages.k8sio-tools
+                    config.packages.goreman
+                    config.packages.stern
+                    config.packages.staticfiles
+                    config.packages.${package.name}
+                    nodePackages.shell.nodeDependencies
+                    gopls
+                    go
+                    jq
+                    nodejs
+                    pythonEnv
+                    clang-tools
+                    protobuf
+                    myyarn
+                  ];
+                  enterShell = ''
+                    ./hack/port-forward.sh;
+                    ./hack/free-port 9090;
+                    ./hack/free-port 2746;
+                    ./hack/free-port 8080;
```

Comment:
you forgot the .sh extension here

Reply:
Are you writing these comments to yourself? 😂

Reply:
Lol yes, I wanted to remind myself to fix this.